### DISCUSSION TITLE
Add a way to add custom fields to the "meta" section of the sidebar

### DISCUSSION THREAD

**Discussion by qbeauperin** [payloadcms/payload#2318] [Feature Requests & Ideas]
Following up on David's thread from Discord, it would be nice to have the option to customize the "meta" section in the sidebar.

![image](https://user-images.githubusercontent.com/1763338/225121140-7e9790cb-e038-4d73-8740-8d9297a2bdc1.png)

The current UI suggests that the top of the sidebar is where inputs and user actions happen, while the bottom is more of a "read-only" section. Some use case examples of what would imo fit better down there:

- Live/Preview urls
- Translation progress/status
- Analytics data snippets (page views, average time on page, etc)

Extending the `admin.position` property might be a way to go about this?

**Comment by richadr** [comment]
It makes sense to add this.

I'm here because I'm looking for a natural place to add a "live" url.

**Reply by qbeauperin** [reply]
I've got a very similar use case with preview links, it would make a lot more sense in terms of UX for these to be in that meta section.
![image](https://github.com/payloadcms/payload/assets/1763338/1e2ff7e7-49bc-4c44-9d25-52ed1f89c531)
